README.md

```diff
@@ -37,10 +38,152 @@ Main features:
       - name: users
         hostname: database1.example.org
         port: 5433
+```
+
+## Example Playbook with service user
```

Comment:
I don't agree with you on this point.
I use Ansible for end-to-end automation.
Your role is older and has many users, so I support the secure approach. However, I would like to give the user how to proceed in an approach where initialization etc. should also be done automatically. 
Suggestion:
- Since more complex scenario move to the end of README.me or 
- I put another file in the root with the example.

Reply:
The README is to give a short intro of the role. You can add your extended examples in a separate file, since we don't have a full docs site for this role.

Reply:
done